### ISSUE TITLE
Training problem

### ISSUE THREAD

**Issue by extraordinaryYouth** [ymxlzgy/echoscene#9]
Please tell me how many epochs you used to train echoscene and how many epochs you used to train echolayout.

Thank you so much!

**Comment by ymxlzgy** [comment]
We train 200 epochs for echolayout and  250 for echoscene. Please check our supp.E.2 for more details.

**Comment by ymxlzgy** [comment]
Sorry, it is been a long period. I think 2000 is the correct number. The training set contains 5,037 rooms, and the bs was 64 when I trained. The learning rate iterates from 1e-4 at step 0 to 5e-6 at 140,000. So the epochs would be near 140,000 / (5,037 / 64), which is approximately 2,000.
